tests/e2e/specs/devconsole-intergration/DevConsoleIntegration.spec.ts

```diff
@@ -0,0 +1,99 @@
+/*********************************************************************
+ * Copyright (c) 2019-2023 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import {
+    SideBarView,
+    ViewItem,
+    ViewSection
+} from 'monaco-page-objects';
+import { registerRunningWorkspace } from '../MochaHooks';
+import { LoginTests } from '../../tests-library/LoginTests';
+import { e2eContainer } from '../../configs/inversify.config';
+import { CLASSES } from '../../configs/inversify.types';
+import { WorkspaceHandlingTests } from '../../tests-library/WorkspaceHandlingTests';
+import { ProjectAndFileTests } from '../../tests-library/ProjectAndFileTests';
+import { Logger } from '../../utils/Logger';
+import { expect } from 'chai';
+import { TestConstants } from '../../constants/TestConstants';
+import { OcpMainPage } from '../../pageobjects/openshift/OcpMainPage';
+import { OcpImportFromGitPage } from '../../pageobjects/openshift/OcpImportFromGitPage';
+import { KubernetesCommandLineToolsExecutor } from '../../utils/KubernetesCommandLineToolsExecutor';
+import { GitUtil } from '../../utils/vsc/GitUtil';
+import { OcpApplicationPage } from '../../pageobjects/openshift/OcpApplicationPage';
+
+const projectAndFileTests: ProjectAndFileTests = e2eContainer.get(CLASSES.ProjectAndFileTests);
+const loginTests: LoginTests = e2eContainer.get(CLASSES.LoginTests);
+const workspaceHandlingTests: WorkspaceHandlingTests = e2eContainer.get(CLASSES.WorkspaceHandlingTests);
+const ocpMainPage: OcpMainPage = e2eContainer.get(CLASSES.OcpMainPage);
+let ocpImportPage: OcpImportFromGitPage;
+let ocpApplicationPage: OcpApplicationPage;
+const kubernetesCommandLineToolsExecutor: KubernetesCommandLineToolsExecutor = new KubernetesCommandLineToolsExecutor();
+
+// works only with no-admin user
+suite(`DevConsole Integration`, async function (): Promise<void> {
+    // test specific data
+    const gitImportRepo: string = 'https://github.com/musienko-maxim/summit-lab-spring-music';
```

Comment:
Is it possible to reuse one of the Eclipse Che devfile sample repo instead of personal repo?

It is also needed to parameterize `gitImportRepo` to make it possible to run the test using Dev Spaces sample.

Reply:
So I used this fork because I fixed an error in the test application (summit-lab-spring-music) in this repo. The easiest way was not to add these changes in the origin repo. I created a fork and used it.  I cannot remember the place with a fix. But anyway you can replace this test project in any place and use it at your discretion

Reply:
@musienko-maxim So should it be exactly this java project or we can use, for example, our NodeJs sample? Which will be labels, in that case :)

Reply:
Or we can just move it to our qe-crw GitHub repo

Reply:
I moved it to our common test repo: https://github.com/crw-qe/summit-lab-spring-music.git

Reply:
thanks